### PR TITLE
Add System Version Check cask

### PR DESCRIPTION
**Pull request by leejongyoung**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/system-version-check.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  version "1.0.0"
+  sha256 "331dae5973a0091552cd888452c220e436fb11aa70f5df5510d772f72db42a7c"
+
+  url "https://github.com/leejongyoung/System-Version-Checker/releases/download/v#{version}/System-Version-Check-v#{version}.dmg"
+  name "System Version Check"
+  desc "Check the version information of your system"
+  homepage "https://github.com/leejongyoung"
+
+  livecheck do
+    url "https://github.com/leejongyoung/System-Version-Checker/releases.atom"
```
Comment (SMillerDev) [inline Casks/system-version-check.rb:11]:
```suggestion
    url :stable
```

**Review by SMillerDev — COMMENTED** [review]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

File: Casks/system-version-check.rb (hunk 1 above)
Comment (leejongyoung) [reply Casks/system-version-check.rb]:
``` shell
$ brew createzap system-version-check
```

## result
No system-version-check settings found.

**Comment by razvanazamfirei** [comment]
Hi, this seems like a nice software, but not notable enough for us to package in Homebrew for now. This is not a decision on the merits of the software, but we cannot distribute every single piece of software and have to draw a line on “notability” somewhere. Of course, we would be happy to reconsider in the future.

We're also unable to accept it at this time because it does not pass our signature verification check that are a requirement for new submissions:

> macOS on ARM requires applications to be signed. Please contact the upstream developer to let them know they should notarize their package.

Please consider hosting this [in your own Tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap) and if the developers sign their app in the future we would then be happy to consider adding this to `homebrew-cask`.

**Pull request closed without merging**